### PR TITLE
Documentation: icon

### LINKED ISSUE #394: Documentation: icon

### PR DESCRIPTION
**Pull request by SergejLust**
By creating this pull request you agree to the terms in CONTRIBUTING.md.
https://github.com/Infineon/.github/blob/master/CONTRIBUTING.md
--- DO NOT DELETE ANYTHING ABOVE THIS LINE ---
- created usage.mdx
- added images
- added documentation

<details>
  <summary>📦 Published PR as canary version: <code>20.4.0--canary.395.a39de7022b0a11b69c939f08c92eb7f798781780.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @infineon/infineon-design-system-stencil@20.4.0--canary.395.a39de7022b0a11b69c939f08c92eb7f798781780.0
  # or 
  yarn add @infineon/infineon-design-system-stencil@20.4.0--canary.395.a39de7022b0a11b69c939f08c92eb7f798781780.0
  ```
</details>

### CODE REVIEW

File: packages/components/src/components/icon/Usage.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
+import { Meta } from '@storybook/blocks';
+
+import * as IconStories from './icon.stories';
```
Comment (verena-ifx) [inline packages/components/src/components/icon/Usage.mdx:4]: here, the correct path is: './infineonIconStencil.stories';

File: packages/components/src/components/icon/Usage.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import IconColorAlignment from '../../../public-storybook/images/icon/icon-color-alignment.png';
+import IconAlignment from '../../../public-storybook/images/icon/icon-alignment.png';
+import IconSizes from '../../../public-storybook/images/icon/icon-sizes.png';
+import IconStructure from '../../../public-storybook/images/icon/icon-structure.png';
+import IconSizeAlignment from '../../../public-storybook/images/icon/icon-size-alignment.png';
+import IconTextColorAlignment from '../../../public-storybook/images/icon/icon-text-color-alignment.png';
```
Comment (verena-ifx) [inline packages/components/src/components/icon/Usage.mdx:10]: this image doesnt exist

**Comment by verena-ifx** [comment]
@SergejLust I have added some comments that should solve the issue of the checks failing

File: packages/components/src/components/icon/Usage.mdx (hunk 1 above)
Comment (SergejLust) [reply packages/components/src/components/icon/Usage.mdx]: Thanks, i've fixed it :)

**Pull request merged**